insights/parsers/tests/test_krb5.py

```diff
@@ -0,0 +1,92 @@
+from insights.parsers import krb5
+from insights.tests import context_wrap
+
+
+KRB5CONFIG = """
+# Configuration snippets may be placed in this directory as well
+includedir /etc/krb5.conf.d/
+
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[realms]
+ dns_lookup_realm = false
+ auth_to_local = RULE:[1:$1@$0](.*@.*EXAMPLE.ORG)s/@.*//
+ default_ccache_name = KEYRING:persistent:%{uid}
+ default_ccache_name2 = KEYRING:%{uid}:persistent
+ EXAMPLE.COM = {
+  kdc = kerberos.example.com
+  admin_server = kerberos.example.com
+ }
+ 89EXAsdcsdMPLE.COM = {
```

Comment:
I would strongly recommend not making up examples, and instead to collaborate with IdM associates and other people who can give you known configuration files that provide the examples you want to test.